dyna_precice.c

```diff
@@ -209,42 +209,43 @@ void dyna_precice(double **cop, ITG *nk, ITG **konp, ITG **ipkonp, char **lakonp
 
   /* Adapter: Put all the CalculiX data that is needed for the coupling into an array */
   struct SimulationData simulationData = {
-      .ialset    = ialset,
-      .ielmat    = ielmat,
-      .istartset = istartset,
-      .iendset   = iendset,
-      .kon       = kon,
-      .ipkon     = ipkon,
-      .lakon     = lakon,
-      .co        = co,
-      .set       = set,
-      .nset      = *nset,
-      .ikboun    = ikboun,
-      .ikforc    = ikforc,
-      .ilboun    = ilboun,
-      .ilforc    = ilforc,
-      .nboun     = *nboun,
-      .nforc     = *nforc,
-      .nelemload = nelemload,
-      .nload     = *nload,
-      .sideload  = sideload,
-      .mt        = mt,
-      .nk        = *nk,
-      .ne        = *ne,
-      .theta     = &theta,
-      .dtheta    = &dtheta,
-      .tper      = tper,
-      .nmethod   = nmethod,
-      .xload     = xload,
-      .xforc     = xforc,
-      .xboun     = xboun,
-      .ntmat_    = ntmat_,
-      .vold      = vold,
-      .veold     = veold,
-      .fn        = fn,
-      .cocon     = cocon,
-      .ncocon    = ncocon,
-      .mi        = mi};
+      .ialset         = ialset,
+      .ielmat         = ielmat,
+      .istartset      = istartset,
+      .iendset        = iendset,
+      .kon            = kon,
+      .ipkon          = ipkon,
+      .lakon          = lakon,
+      .co             = co,
+      .set            = set,
+      .nset           = *nset,
+      .ikboun         = ikboun,
+      .ikforc         = ikforc,
+      .ilboun         = ilboun,
+      .ilforc         = ilforc,
+      .nboun          = *nboun,
+      .nforc          = *nforc,
+      .nelemload      = nelemload,
+      .nload          = *nload,
+      .sideload       = sideload,
+      .mt             = mt,
+      .nk             = *nk,
+      .ne             = *ne,
+      .theta          = &theta,
+      .dtheta         = &dtheta,
+      .tper           = tper,
+      .nmethod        = nmethod,
+      .xload          = xload,
+      .xforc          = xforc,
+      .xboun          = xboun,
+      .ntmat_         = ntmat_,
+      .vold           = vold,
+      .veold          = veold,
+      .fn             = fn,
+      .cocon          = cocon,
+      .ncocon         = ncocon,
+      .mi             = mi,
+      .isModalDynamic = 1};
```

Comment:
Why so many changed lines? Is this just due to formatting? Looks ok, but just wondering.

Reply:
Yes, because the latest addition has a much longer name. Otherwise it's unchanged.